jerry-core/ecma/builtin-objects/ecma-builtin-arraybuffer-prototype.inc.h

```diff
@@ -0,0 +1,66 @@
+/* Copyright 2016 Intel Corporation
```

Comment:
Please use the generic copyright notice for the project instead, we are phasing out individual copyright notices:  
"Copyright JS Foundation and other contributors, http://js.foundation" (without the quotes)  
There's more information about that in #1473.